src/Packagist/WebBundle/Resources/public/css/main.css

```diff
@@ -991,6 +991,21 @@ ul.packages .metadata-block:last-child {
   line-height: 100%;
 }
 
+.wrapper .content .package .readme table {
+  margin: 0 0 15px;
+}
+.wrapper .content .package .readme th,
+.wrapper .content .package .readme td {
+  border: 1px solid #CCC;
+  padding: 5px 10px;
+}
+.wrapper .content .package .readme th {
+  background: #EEE;
+}
+.wrapper .content .package .readme table code {
```

Comment:
do we need such a high specificity ? couldn't we use ``.readme table code`` only (and same for all selectors) ?